packages/open-payments/src/client/incoming-payment.test.ts

```diff
@@ -94,6 +111,106 @@ describe('incoming-payment', (): void => {
     })
   })
 
+  describe('createIncomingPayment', (): void => {
+    test.each`
+      incomingAmount                                      | expiresAt                                      | description  | externalRef
+      ${undefined}                                        | ${undefined}                                   | ${undefined} | ${undefined}
+      ${{ assetCode: 'USD', assetScale: 2, value: '10' }} | ${new Date(Date.now() + 60_000).toISOString()} | ${'Invoice'} | ${'#INV-1'}
+    `(
+      'returns the incoming payment on success',
+      async ({
+        incomingAmount,
+        expiresAt,
+        description,
+        externalRef
+      }): Promise<void> => {
+        const incomingPayment = mockIncomingPayment({
+          incomingAmount,
```

Comment:
`mockIncomingPayment` always defines `incomingAmount`
https://github.com/interledger/rafiki/blob/465b537fe19a417ce45a79fe90cdefb88f661b6c/packages/open-payments/src/test/helpers.ts#L55-L59
At first I was going to suggest only define `incomingAmount` in `mockIncomingPayment` if it specified in the `overrides`.
Now I'm wondering if it should actually be allowed for a created incoming payment to have an `incomingAmount` if it isn't specified in the request? Or if it _is_ specified, should the created incoming payment be required to have that `incomingAmount` value?
I'm surprised neither of two have been mentioned in:
- https://github.com/interledger/rafiki/issues/649

Reply:
I don't think it's possible in OpenAPI 3.1's JSON Schemsa 2020-12 to validate the presence or value of response's `incomingAmounnt` based on the that of the request.

I think we should at least validate that if `incomingPayment` is specified, it is included in the response (with an equal or lesser value?)

Reply:
`incomingPayment.value` should be the same in the request and the response, so we could just check for equality.

But in line with what was said previously, we can merge this in and then follow up with the extra validation